components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1549,6 +1563,10 @@ void RewardsServiceImpl::OnTipsUpdatedData(const ledger::PublisherInfoList list)
   }
 }
 
+void RewardsServiceImpl::AddRecurringPayment(const std::string& publisher_key, double new_amount) {
```

Comment:
can you just use `SaveRecurringDonation` directly, so that we don't create new function?

Reply:
done